src/coreclr/src/jit/emitarm64.cpp

```diff
@@ -12906,6 +12908,20 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
                     result.insLatency    = PERFSCORE_LATENCY_2C;
                     break;
 
+                case INS_trn1:
+                case INS_trn2:
+                    if ((ins == INS_trn1 || ins == INS_trn1) && (id->idInsOpt() == INS_OPTS_2D))
```

Comment:
you are doing the same comparison twice, its also not clear why the comparison here is needed given the above cases.

Reply:
ah, the rebasing did have a casualty.. I'll fix that up.

Reply:
ah, the rebasing did have a casualty.. I'll fix that up. The if was because there was a fall through before.